### PR TITLE
update global search and tenant events guide to use sdk scrolling

### PR DESCRIPTION
**Pull request by bendoyon**

### CODE REVIEW

File: docs/guides/tenant-events.mdx [hunk 1]
```diff
@@ -52,43 +47,30 @@ if not api_key:
...
-    resp.raise_for_status()
+):
+    # Rate limiting.
+    time.sleep(1)
+
     resp_data: dict = resp.json()
 
     fetched_pages += 1
-    num_results: int = len(resp_data["items"])
+    num_results: int = len(resp_data["items"] or [])
```
Comment (aviau) [inline docs/guides/tenant-events.mdx:69]: Items isn`t always present? why not use .get()?

File: docs/guides/tenant-events.mdx [hunk 2]
```diff
@@ -19,21 +19,16 @@ of all results.
...
 [Flare standard paging pattern <Icon icon="book" size={16} />](/concepts/paging).
 
-Differences:
-- The `next` *response field* is instead named `search_after`.
-- The `from` *request parameter* is instead named `search_after`.
-
-Other than the naming difference, the function is the same:
-- Use the last response's `search_after` field as the value for the following request's `search_after` parameter.
-- Stop when the returned `search_after` is empty.
+- Use the last response's `next` field as the value for the following request's `next` parameter.
```
Comment (aviau) [inline docs/guides/tenant-events.mdx:25]: Remove these lines, they no longer add any information now that we linked to paging

File: docs/guides/global-search.mdx [hunk 3]
```diff
@@ -30,22 +30,17 @@ To achieve the desired results, the following parameters will be used:
...
 [Flare standard paging pattern <Icon icon="book" size={16} />](/concepts/paging).
 
-Differences:
-- The `next` *response field* is instead named `search_after`.
-- The `from` *request parameter* is instead named `search_after`.
-
-Other than the naming difference, the function is the same:
-- Use the last response's `search_after` field as the value for the following request's `search_after` parameter.
-- Stop when the returned `search_after` is empty.
+- Use the last response's `next` field as the value for the following request's `next` parameter.
```
Comment (aviau) [inline docs/guides/global-search.mdx:36]: I'd remove these two lines

File: docs/guides/global-search.mdx [hunk 4]
```diff
@@ -72,46 +67,33 @@ from_timestamp: str = (
...
-    # Fetch the next page
-    resp = api_client.get(
-        "/firework/v2/search/",
-        params=params,
-    )
-    resp.raise_for_status()
+        "from": None,
+    },
+):
+    # Rate limiting.
```
Comment (aviau) [inline docs/guides/global-search.mdx:86]: rate limit a the end of the loop?

**Review by aviau — APPROVED** [review]

File: docs/guides/tenant-events.mdx (hunk 1 above)
Comment (bendoyon) [reply docs/guides/tenant-events.mdx]: Linting would complain that the value passed to len was not Sized, but it seems to be fine now?

**Pull request merged**